charon/src/translate_functions_to_im.rs

```diff
@@ -545,374 +545,341 @@ fn translate_projection<'tcx>(
     return (projection, path_type);
 }
 
-/// Translate some cases of a constant operand.
-///
-/// A constant operand is usually an instance of a primitive type (bool, char,
-/// integer...). However, it may also be an instance of a degenerate ADT or
-/// tuple (if an ADT has only one variant and no fields, it is a constant,
-/// and unit is encoded by MIR as a 0-tuple). This function handles the
-/// primitive type cases: the other cases should have been filtered and
-/// handled elsewhere.
-fn translate_operand_constant_value_constant_value<'tcx>(
-    ty: &Ty<'tcx>,
-    value: &mir::interpret::ConstValue<'tcx>,
-) -> (ty::ETy, v::ConstantValue) {
-    trace!();
-    match value {
-        mir::interpret::ConstValue::Scalar(scalar) => {
-            // The documentation explicitly says not to match on a
-            // scalar. We match on the type and convert the value
-            // following this, by calling the appropriate `to_*`
-            // method
```

Comment:
This comment is important (move it to `translate_constant_integer_like_value`)

Reply:
Yup, done